presto-docs/src/main/sphinx/release/release-0.227.rst

```diff
@@ -0,0 +1,39 @@
+=============
+Release 0.227
+=============
+
+General Changes
+_______________
+* Enable spatial joins (broadcast and distributed) for ST_Equals, ST_Overlaps, ST_Crosses, and ST_Touches.
```

Comment:
What does "enable" means. Is that a configuration change (which should not be in a release notes), or else it should be `Add spatial join support for ...`

Reply:
Functions should be reference properly:
```
:func:`ST_Equals`
```
as stated in the [Release Notes Guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#formatting)

Reply:
@caithagoras It was not a configuration change. There're different types of operations that can be used in spatial joins, e.g.  Contains, Within, Equals, Overlaps, Crosses, Touches, and Distance. Previously Equals, Overlaps, Crosses and Touches were not supported. https://github.com/prestodb/presto/pull/13460 "enables" these operations for spatial joins. How about this:
`* Add spatial join (broadcast and distributed) support for :func:`ST_Equals`, :func:`ST_Overlaps`, :func:`ST_Crosses`, and :func:`ST_Touches`.`